npm-scripts.mjs

```diff
@@ -128,6 +129,12 @@ async function run() {
 			break;
 		}
 
+		case 'worker:build-lib': {
+			buildWorkerLib();
+
+			break;
+		}
```

Comment:
Is there any real reason to add this as a public script in `package.json`? I removed many of these in the past because it didn't make any sense to make them public. Does it make sense that we call `npm run worker:build-lib` directly? And why is this about "worker build lib"?

Reply:
leftover.